common/presenters/table/object-to-table-head.js

```diff
@@ -1,15 +1,45 @@
 const { pickBy } = require('lodash')
+const queryString = require('query-string')
 
 const i18n = require('../../../config/i18n')
 
+function buildUrl(head) {
+  const {
+    currentUrl: { pathname, query },
+    key,
+    currentSort,
+  } = head
+  return queryString.stringifyUrl({
+    url: pathname,
+    query: {
+      ...query,
+      sortBy: key,
+      sortDirection: currentSort,
+    },
+  })
+}
+
+function buildWrapper(head) {
+  const label = i18n.t(head.html)
+  const url = buildUrl(head)
+  let sort = 'none'
```

Comment:
Having a quick read up on [`aria-sort`](https://www.digitala11y.com/aria-sort-properties/) it seems it should be applied to the `th` element rather than the anchor element.

It also looks like if no attribute is provided the default value is `none` so we can probably just drop this attribute on all other headers and just apply it with the relevant sort to the column that the table is sorted by.

Reply:
Yeah, the html is all over the place. I wanted it to work with the `sortable-table` styles which are very different: it's all client-side so there are no links to speak of. I just wanted to see the arrows, so I hammered it in. This is not meant to be the solution, just a stepping stone. It's likely that `aria-sort` must be applied to a `th`, which is not possible afaik without overriding the govuk-table.

One thing I would like to discuss is the possibility of just copying / adapting the `sortable-table` styles instead of importing them. This would make things much simpler.

Reply:
I have realised I can pass `aria-sort` to the th through attributes. Still doesn't solve the `button` issue, but it's a step in the right direction.